game-core/src/main/java/games/strategy/debug/DefaultConsole.java

```diff
@@ -0,0 +1,158 @@
+package games.strategy.debug;
```

Comment:
Because I copied this file from _Console.java_ instead of renaming it, Git diff renders it as a completely new file, making the changes harder to spot.  In reality, this file is like 99% identical to the original _Console.java_.  The actual changes are below:

```diff
diff --git a/home/ssoloff/Downloads/Console.java b/game-core/src/main/java/games/strategy/debug/DefaultConsole.java
index 9f767c431..b140c9054 100644
--- a/home/ssoloff/Downloads/Console.java
+++ b/game-core/src/main/java/games/strategy/debug/DefaultConsole.java
@@ -5,7 +5,6 @@ import java.awt.Toolkit;
 import java.awt.datatransfer.StringSelection;
 import java.awt.event.ItemEvent;
 import java.util.logging.Level;
-import java.util.logging.LogManager;
 
 import javax.swing.AbstractButton;
 import javax.swing.Action;
@@ -34,20 +33,18 @@ import games.strategy.ui.SwingComponents;
 import lombok.AllArgsConstructor;
 import lombok.extern.java.Log;
 
-/**
- * A 'console' window to display log messages to users.
- */
 @Log
-public final class Console {
+final class DefaultConsole implements Console {
   private static final ImmutableCollection<LogLevelItem> LOG_LEVEL_ITEMS = ImmutableList.of(
       new LogLevelItem("Errors and Warnings", Level.WARNING),
+      new LogLevelItem("Errors, Warnings and Informational Messages", Level.INFO),
       new LogLevelItem("All Messages", Level.ALL));
 
   private final JTextArea textArea = new JTextArea(20, 50);
   private final JFrame frame = new JFrame("TripleA Console");
   private Level logLevel = Level.WARNING;
 
-  public Console() {
+  DefaultConsole() {
     setLogLevel(getDefaultLogLevel());
 
     ClientSetting.showConsole.addListener(gameSetting -> {
@@ -98,11 +95,7 @@ public final class Console {
 
   private void setLogLevel(final Level level) {
     logLevel = level;
-    LogManager.getLogManager().getLogger("").setLevel(attenuateLogLevel(level));
-  }
-
-  private static Level attenuateLogLevel(final Level level) {
-    return (level.intValue() < Level.INFO.intValue()) ? Level.INFO : level;
+    LoggerManager.setLogLevel(level);
   }
 
   private AbstractButton newLogLevelButton() {
@@ -147,18 +140,16 @@ public final class Console {
     ClientSetting.loggingVerbosity.setValueAndFlush(level.getName());
   }
 
+  @Override
   public void setVisible(final boolean visible) {
     frame.setVisible(visible);
   }
 
+  @Override
   public void append(final String s) {
     SwingUtilities.invokeLater(() -> textArea.append(s));
   }
 
-  public void appendLn(final String s) {
-    append(s + "\n");
-  }
-
   @AllArgsConstructor
   private static final class LogLevelItem {
     final String label;
```